transcript/src/main/java/com/transcriptanalyzer/transcript/User_Requests_Intents/Service/TranscriptService.java

```diff
@@ -23,21 +24,33 @@ public class TranscriptService {
 
     private ApiRepository apiRepository;
 
-//    //Stores the APIKey in the apiAccess.properties files
-//    //Switched to static
-//    public static void storeAPIKey(String apiKey) {
-//    }
-//
-//
-//    //Stores the APIVersion in the apiAccess.properties files
-//    //Switched to static
-//    public static void storeAPIVersion(String apiVersion) {
-//
-//    }
+    public static ArrayList<String> getIntents() throws IOException {
+        ArrayList<String> list = new ArrayList<>();
+    //Intent 1
+        String intent_1 = getJSONContent().get(0).get(1).get(0).replaceAll("[:{\"}]","");
+        intent_1 = intent_1.substring(11);
```

Comment:
Also, why do we need substring(11). I think we can make our code much easier to follow with more comments/documentation.